### DISCUSSION TITLE
Styling frameworks to use outside of Monaco editor component

### DISCUSSION THREAD

**Discussion by TheDanielDoyle** [alex-hedley/BlazorInteractive#2] [General]
We have a plethora of CSS frameworks to choose from.

### Blazor Components

- [Blazorise](https://blazorise.com/docs)
- [Radzen](https://blazor.radzen.com/)

### CSS

- [Bootstrap](https://getbootstrap.com/)
- [Bulma](https://bulma.io/)
- [Foundation](https://get.foundation/)
- [Materialize](https://materializecss.com/)
- [Material Framework](https://nt1m.github.io/material-framework/)
- [Pico](https://picocss.com/)
- [Primer](https://primer.style/)
- [Pure](https://purecss.io/)
- [Semantic UI](https://semantic-ui.com/)
- [Siimple](https://siimple.xyz/)
- [Tailwind](https://tailwindcss.com/)
- [UIKit](https://getuikit.com/)

**Comment by AlexHedley** [comment]
- fluentui
  - https://baaijte.net/blog/microsoft-fast-components-fluentui-2.2/

**Comment by AlexHedley** [comment]
https://twitter.com/chbernasconic/status/1653762432060342275

https://www.youtube.com/watch?v=bsu0cCjeVaw

**Comment by AlexHedley** [comment]
mudblazor

- https://mudblazor.com
